src/service/performance-impl.js

```diff
@@ -302,6 +303,10 @@ export class Performance {
       return this.enabledExperiments_;
     }
     const experiments = [];
+    // Add RTV version as experiment ID, so we can slice the data by version.
+    if (getMode(this.win).rtvVersion) {
+      experiments.push(getMode(this.win).rtvVersion);
```

Comment:
i think we probably stick with ampexp. we are not supposed to introduce arbitrary new param to CSI (which is shared by many other Google products).